Documentation/warnings.rst

```diff
@@ -0,0 +1,30 @@
+Cilium Warnings
+###############
+
+..
+   NOTES: each warning has an id (e.g., cgrouplowproccount) used in the code to
+   refer to specifics in this document. Code points to the latest version of the
+   docs, so text should be valid for all supported Cilium versions.
+
+.. _cgrouplowproccount:
+
+Low cgroup process count
+------------------------
+
+For :ref:`host-services`, Cilium attaches eBPF programs at appropriate cgroup
+hooks. For this to work properly, the cgroup that the programs are attached
+needs to be in a right level at the cgroup hierarchy so that all pods are
```

Comment:
```suggestion
hooks. For this to work properly, the cgroup that the programs are attached to
needs to be at the right level in the cgroup hierarchy so that all pods are
```